src/Microsoft.SqlTools.ServiceLayer/Profiler/ProfilerSession.cs

```diff
@@ -24,10 +24,7 @@ public class ProfilerSession
         private TimeSpan pollingDelay = DefaultPollingDelay;
         private ProfilerEvent lastSeenEvent = null;
 
-        /// <summary>
-        /// Unique ID for the session
-        /// </summary>
-        public string SessionId { get; set; }
+        // TODO: Add ability to trigger instant polling
```

Comment:
What does this mean? Is there an issue tracking it or is it something you meant to include in this PR?

Reply:
Sorry, these were meant to be more personal notes for myself on features that I'd need to add before 6/20 and where they should go, I'll go back and remove the extraneous TODO's. They're all tracked in one profiler issue that lists all my remaining tasks for the profiler.